cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/SQLConn.scala

```diff
@@ -0,0 +1,100 @@
+package edu.uci.ics.cloudberry.zion.model.impl
+import edu.uci.ics.cloudberry.zion.model.datastore.IDataConn
+
+import play.api.libs.ws.WSResponse
+import play.api.libs.json.Json
+import play.api.libs.json._
+
+import scala.concurrent.{ExecutionContext, Future}
+
+import java.sql.{Connection, DriverManager}
+import java.io.InputStream
+import java.util.Date
+import java.text.SimpleDateFormat
+
+class SQLConn(url: String)(implicit ec: ExecutionContext) extends IDataConn {
+  val driver = "com.mysql.jdbc.Driver"
+  val username = "root"  //TODO: how to config?
+  val password = "2048"
+  var connection:Connection = _
+  Class.forName(driver)
+  connection = DriverManager.getConnection(url, username, password)
+  val stream: InputStream = getClass.getResourceAsStream("/ddl/berry.json")
```

Comment:
Maybe we still do? When starting up and berry.meta is empty, we should return a schema first? I've tried other ways but failed.